ocrd-tool.json

```diff
@@ -45,9 +45,9 @@
           ]
         },
         "ext": {
-          "description": "Output extension",
+          "description": "Output extension. Set to empty string to derive extension from mimetype.",
```

Comment:
According to https://en.wikipedia.org/wiki/Media_type, the new term is _media type_. Formerly it was _MIME type_.